android/app/src/main/res/layout/activity_first_time.xml

```diff
@@ -6,43 +6,37 @@
     android:layout_height="match_parent"
     tools:context="io.peggyjo.peggymeter.FirstTimeActivity">
 
-    <LinearLayout
+    <GridLayout
```

Comment:
Why change LinearLayout to GridLayout?

Reply:
Grid layout is good to create grid base interfaces that looks pretty (in both album and normal mode). Previous usage of linear layout was mostly a hack to demonstrate welcome screen.